python_ta/reporters/templates/template.html

```diff
@@ -8,138 +8,182 @@
-                            {% set occurrences = messages[0][message_id] %}
-                        <div class="error-instance">
-                            <div class="info">
-                                <span class="error-link code-error-id">
-                                    <a href="http://www.cs.toronto.edu/~david/pyta/&#35;{{message_id}}" target="_blank">&#10064; {{ message_id }} ({{ occurrences[0].symbol }})</a>
-                                </span>
-                                {% set num_occurrences = occurrences|length %}
-                                <span class="occurrences">
-                                {{ num_occurrences }}
-                                {% if num_occurrences == 1 %}
-                                 occurrence.
-                                {% else %}
-                                 occurrences.
-                                {% endif %}
-                                </span>
-                                <span class="shown">
-                                {% if limit < num_occurrences %}
-                                 (First {{ limit }} shown).
-                                {% endif %}
-                                </span>
-                                <span class="slider">
-                                    <button> Expand/Collapse </button>
-                                </span>
-                            </div>
-                            <div class="message-container collapsible">
-                                {% for indiv in occurrences %}
-                                <div class="message">
-                                    <h4 class="message-name">
-                                        [Line {{ indiv.line }}] {{ indiv.msg }}
-                                        <span class="slider">
-                                            <button style="float: right; background: #458B00;"> Expand/Collapse </button>
-                                        </span>
-                                    </h4>
-                                    {% if indiv.snippet != '' %}
-                                    <div class="collapsible">
-                                      <code>{{ indiv.snippet }}</code>
-                                    </div>
+        <div class="main-grid">
+            <header style="grid-column: 1 / span 2">
+                <h1 class="title">PyTA Error Report</h1>
+                <time>{{ date_time }}</time>
+            </header>
+            <aside style="position: sticky">
+                <section id="sidebar">
+                    <article class="section-heading"><h2>Table of Contents</h2></article>
+                    <article>
+                        <ul>
+                            {% for filename, messages in grouped_messages.items() %}
+                            <li>
+                                <a style="color: DimGray">
+                                    {{filename.split('\\')[-1]}}
```

Comment:
This splitting won't work on Macs. You can use a Python built-in function from `os` to get the filename.